Resources/doc/config.rst

```diff
@@ -343,17 +343,13 @@ following syntax:
             </doctrine_mongodb:config>
         </container>
 
-Now you can retrieve the configured services connection services:
-
-.. code-block:: php
```

Comment:
Why is this a bad thing?

Reply:
It’s not called but more noise then you need and it reads more fluently.
Also you have highlighted code block in PHPStorm for example

Reply:
Ah ok, I thought it was the `php` part that would allow IDEs to understand there's PHP (not brainfuck for one) inside the code block